lib/heuristics/concerns/periodic_data_initialisation.rb

```diff
@@ -207,7 +207,7 @@ def collect_indices(vrp)
     }
 
     @indices.each_key{ |point_id|
-      @points_assignment[point_id] = { vehicles: [], days: [], maximum_visits_number: 0 }
+      @points_assignment[point_id] = { vehicles: [], days: [], services_ids: [] }
```

Comment:
Our convention for an array of ids is `<singular name of object>_ids`
```suggestion
      @points_assignment[point_id] = { vehicles: [], days: [], service_ids: [] }
```

Reply:
It seems you've missed this one